Added section on beta testers (#3165)
PostHog/posthog.com: contents/handbook/product/user-feedback.md

> 😍 Want to share feedback? [File an issue](https://posthog.com/posthog/posthog/issues) or reach out on [Slack](/slack). **We're always happy to hear from you!**

### Beta Testers

> We have a group in our community slack called #beta-testers this is a group of engaged community users and paying customers who are keen to give feedback on our product in order to improve the product for themselves and other users.

Reviewer: We also have a cohort of "Beta Users" (we may want to rename for consistency) which contain users who have explicitly expressed interest in being included in beta features. In the feature release process, it makes sense to release to them too.

Author: Yep - agreed, ideally we should try and seed the slack group with the folks from the Beta Users group. Would you be able to help with that @paolodamico ?

> We have a group in our community slack called #beta-testers this is a group of engaged community users and paying customers who are keen to give feedback on our product in order to improve the product for themselves and other users.

Reviewer 2: ```suggestion
We have a group in our [community Slack](/slack) called #beta-testers for engaged community users and paying customers who are keen to give feedback on our product in order to improve it for themselves and other users.
```

> If you would like to get feedback on a feature you're building, share details of the feature and the feedback you'd like to receive in with this group. Be mindful that we shouldn't overburden the group with lots of small changes and focus on getting the feedback we really need to make better decisions. We should also follow up with the beta-testers later on to show them how their feedback has helped build a better product.

Reviewer: Perhaps something we'll need to iterate more, but in my experience posting in a channel will lead to less optimal results than individual reach out. If we could keep the context on what users are generally interested in (e.g. we know Weyert cares about feature flags and/or experimentation) and share that context throughout our team, we'll make this process yield better results.

> If you would like to get feedback on a feature you're building, share details of the feature and the feedback you'd like to receive in with this group. Be mindful that we shouldn't overburden the group with lots of small changes and focus on getting the feedback we really need to make better decisions. We should also follow up with the beta-testers later on to show them how their feedback has helped build a better product.

Reviewer 2: ```suggestion
If you would like to get feedback on something you're building, share details and the feedback you'd like to receive with this group. Be mindful that we shouldn't overburden the group with lots of small changes. Instead, focus on getting feedback we really need to make better decisions. We should also follow up with the beta-testers later on to show them how their feedback has helped build a better product.
```
